src/redux/Books/bookSlice.js

```diff
@@ -1,18 +1,18 @@
 import { createReducer } from '@reduxjs/toolkit';
 
-export const ADD_BOOK = 'BookStore-App/Books/Add_Book';
-export const REMOVE_BOOK = 'BookStore-App/Books/Remove_Book';
+export const ADDNEW_BOOK = 'BookStore-App/Books/Add_Book';
+export const REMOVEA_BOOK = 'BookStore-App/Books/Remove_Book';
 
 const initialState = {
   books: [],
 };
 
 const bookReducer = createReducer(initialState, (construct) => {
   construct
-    .addCase(ADD_BOOK, (state, action) => {
+    .addCase(ADDNEW_BOOK, (state, action) => {
       state.books.push(action.payload);
     })
-    .addCase(REMOVE_BOOK, (state, action) => {
+    .addCase(REMOVEA_BOOK, (state, action) => {
       state.books.filter((book) => book.id !== action.payload);
     });
   construct.addDefaultCase((state) => state);
```

Comment:
If you were using the `createSlice`, your set-up would have two reducers called addbook and removeBook. These reducers will be exported in order to help your components consume them, Kindly add this to stay consistent with the milestone requirement. You checkout this [example](https://redux-toolkit.js.org/api/createSlice#:~:text=import%20%7B%20createSlice,counterSlice.reducer)